Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -73,23 +73,61 @@ def parser(cls, parser, loggers=None):
         )
 
     @classmethod
-    def get_issue_info(cls, args, repository, **kwargs):
-        # Can give either a bug URL or a title of the new issue.
+    def get_commit_info(cls, args, repository, **kwargs):
+        commit_objects = []
+        commits_to_revert = []
+        commit_bugs = []
+        for c in args.commit:
+            try:
+                commit = repository.find(c, include_log=True)
+            except (local.Scm.Exception, ValueError) as exception:
+                sys.stderr.write('Could not find "{}"'.format(c) + '\n')
+                return None, None, None
+            commit_objects.append(commit)
+            commits_to_revert.append(commit.hash)
+            commit_bugs += [Tracker.from_string(i.link) for i in commit.issues if isinstance(Tracker.from_string(i.link).tracker, bugzilla.Tracker)]
```

Comment:
I would de-duplicate these bugs. It's going to be pretty common to do a multi-revert of a group of commits which reference the same set of bugs.